optuna_integration/_lightgbm_tuner/optimize.py

```diff
@@ -378,13 +378,23 @@ def __init__(
         kwargs: dict[str, Any] = dict(
             num_boost_round=num_boost_round,
             feval=feval,
-            feature_name=feature_name,
-            categorical_feature=categorical_feature,
             callbacks=callbacks,
             time_budget=time_budget,
             sample_size=sample_size,
             show_progress_bar=show_progress_bar,
         )
+
+        deprecated_arg_warning = (
+            "Support for lgb.cv with argument {deprecated_arg} was removed from lightgbm 4.6.0 "
+            "and will not be supported by optuna in the future."
+        )
```

Comment:
It would be great to add explanations in the docstring. Could you add them like [this](https://github.com/optuna/optuna/blob/f03b4d26ebaf5a5340f8e6b9af02ecdd568b5111/optuna/study/_study_summary.py#L42)? The removed version will be `v6.0.0`.

Reply:
yup!

Reply:
Please see [7a69be9](https://github.com/optuna/optuna-integration/pull/207/commits/7a69be9892065e4ac1f860f40fa2a09674f8b050)